test/core/experimental/vectors/test_vector.py

```diff
@@ -35,7 +35,8 @@ def test_init() -> None:
         VectorSymbol(name, dim, norm=symbols.force, display_latex=latex)
 
     one_newton_force = VectorSymbol(name, dim, norm=1 * units.newton, display_latex=latex)
-    assert_equal(one_newton_force.norm, 1 * units.newton)
+    if one_newton_force.norm:  # mypy complains otherwise
```

Comment:
It's bad idea to silence mypy like this. Now you can skip the assertion check in test if norm is None or false. Use `norm()` or assert norm is not None instead of `if`